.github/workflows/release.yml

```diff
@@ -126,6 +126,7 @@ jobs:
         uses: ./.github/actions/git-tag
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
+          branch_name: $BRANCH_NAME
```

Comment:
here you need to use `${{ env.BRANCH_NAME }}` since it's not a run command :D

Reply:
Damn :D at least now its clear for me thank you :D I must have misunderstood the docs

Reply:
Fixed :D